presidio-anonymizer/presidio_anonymizer/batch_anonymizer_engine.py

```diff
@@ -0,0 +1,78 @@
+from typing import List, Dict, Union, Iterable
+import collections
+
+from presidio_anonymizer.entities import DictRecognizerResult
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import EngineResult, RecognizerResult
+
+
+class BatchAnonymizerEngine(AnonymizerEngine):

```

Comment:
any reason for inheritance here? Composition over inheritance?

Reply:
Good question. Do we think this class will evolve in the future to support more batch use cases? If yes, maybe inheritance is better, but I don't know.

Reply:
right now i didn't change this since i saw that the other anonymizer uses inheritance as well

let me know if you think it would be better to change this

Reply:
> right now i didn't change this since i saw that the other anonymizer uses inheritance as well
> 
> let me know if you think it would be better to change this

Which other anonymizer? I don't see anything else inheriting from AnonymizerEngine. Anyway I think that since its not a classic 'BatchAnonymizerEngine' is type of 'AnonymizerEngine' but just uses it - composition might be better

Reply:
I misread the deanonymize_engine
changed to composition instead